api/utils/index.js

```diff
@@ -130,13 +130,8 @@ function mapValues(fn, obj) {
 }
 
 function wrapHandler(fn) {
-  return (...args) => fn(...args).catch(args[1].error);
-  // We really want to just call `next` (args[2]) with the error
-  // but we currently have this other error handling that is short
-  // circuiting the typical Express error handling stack. Save
-  // refactoring it for another day.
-  // This will call `res.error(err)`, see logic in api/responses.
-  // return (...args) => fn(...args).catch(args[2]);
+  // calls `next(error)`
+  return (...args) => fn(...args).catch(args[2]);
```

Comment:
Let's go for it and call `next` to we use the normal express error handling stack. Users of `wrapHandler` and `wrapHandlers` should be aware of this when using.